tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -228,6 +230,8 @@ def _worker(self, idx, node):
                     elif name == "records_consumed":
                         handler.handle_records_consumed(event)
                         self._update_global_position(event, node)
+                    elif name == "record_data":
+                        self.on_record_consumed(event, node)
```

Comment:
Do we need to check if `on_record_consumed` is defined?

Reply:
Wouldn't hurt. I was relying on the --verbose flag not being provided if `on_record_consumed` is not defined, but maybe that's a little brittle.